src/project/internal/projectuiactions.cpp

```diff
@@ -31,91 +31,93 @@ const UiActionList ProjectUiActions::m_actions = {
              mu::context::UiCtxAny,
              mu::context::CTX_ANY,
              TranslatableString("action", "&Open…"),
-             TranslatableString("action", "Open…"),
+             TranslatableString("action", "Open project"),
```

Comment:
@bkunda, what do you think about this change to the string displayed in Preferences?